Dockerfile_worker

```diff
@@ -46,9 +46,6 @@ COPY autopkgtest-wrapper /usr/local/bin/autopkgtest-wrapper
 
 ENV AUTOPKGTEST=/usr/local/bin/autopkgtest-wrapper
 
-# In-case: --new-port 9820
-EXPOSE 9820
-
 # In-case: [PORT] (there isn't a default defined port)
 EXPOSE 9821
```

Comment:
This probably should specify --port=9821 as well

Reply:
Done.